pymc3/distributions/discrete.py

```diff
@@ -380,6 +442,32 @@ def logp(self, value):
                 tt.switch(value, tt.log(p), tt.log(1 - p)), value >= 0, value <= 1, p >= 0, p <= 1
             )
 
+    def logcdf(self, value):
+        """
+        Compute the log of the cumulative distribution function for Bernoulli distribution
+        at the specified value.
+        Parameters
+        ----------
+        value: numeric
```

Comment:
Should probably add the types for multiple values then. Something like the following?
```suggestion
        value: numeric or np.ndarray or theano.tensor
```